build_debian.sh

```diff
@@ -202,7 +197,11 @@ sudo LANG=C chroot $FILESYSTEM_ROOT apt-get -y install apt-transport-https \
                                                        curl \
                                                        gnupg2 \
                                                        software-properties-common
-sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT curl -o /tmp/docker.gpg -fsSL https://download.docker.com/linux/debian/gpg
+if [[ $CONFIGURED_ARCH == armhf ]]; then
+    sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT curl -k -o /tmp/docker.gpg -fsSL https://download.docker.com/linux/debian/gpg
```

Comment:
update-ca-certificates doesnt not solve the issue here.

Reply:
Add above this line

+sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT update-ca-certificates --fresh